dotcom-rendering/src/web/components/ArticleHeadline.tsx

```diff
@@ -689,6 +689,39 @@ export const ArticleHeadline = ({
 							)}
 						</div>
 					);
+				case ArticleDesign.Explainer:
```

Comment:
Does this one need a new branch? Is the return code for feature/review etc. the same as this?

E.g. Could we do this:

```
switch (format.design) {
				case ArticleDesign.Review:
				case ArticleDesign.Recipe:
				case ArticleDesign.Explainer:
				case ArticleDesign.Feature:
					return (
						<div
```

This is just a guess, I could be wrong!

Reply:
Great point, I wasn't sure which of the existing switch branches was appropriate for Explainer. This seems to be good. Just fixed it 👍 

The only point worth mentioning is that, for Analysis we are now having a different font style when theme is labs.  but now for Explainer, we don't apply labs font styles any more